include/itkOMEZarrNGFFImageIO.h

```diff
@@ -231,6 +239,11 @@ class IOOMEZarrNGFF_EXPORT OMEZarrNGFFImageIO : public ImageIOBase
   int                m_TimeIndex = INVALID_INDEX;
   int                m_ChannelIndex = INVALID_INDEX;
   AxesCollectionType m_StoreAxes;
+
+  const static unsigned m_EmptyZipSize = 22;
+  char                  m_EmptyZip[m_EmptyZipSize] = "PK\x05\x06"; // the rest is filled with zeroes
```

Comment:
How is this derived? Required header for zip file format? Could a descriptive comment please be added?